libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClientSettings.java

```diff
@@ -67,6 +70,10 @@ HttpUrl getBaseUrl() {
     return baseUrl;
   }
 
+  void setBaseUrl(String eventsHost) {
```

Comment:
is there a way we could achieve the same with the builder? it feels like this breaks immutability of the `TelemetryClientSettings` class.